app/models/mdm/tag.rb

```diff
@@ -82,11 +85,11 @@ class Mdm::Tag < ActiveRecord::Base
   validates :desc,
             :length => {
                 :maximum => ((8 * (2 ** 10)) - 1),
-                :message => "desc must be less than 8k."
+                :message => TAG_LENGTH_VALIDATION_MSG
```

Comment:
Validation messages should go in `en.yml` instead of using `:message` and constants in the class.